index.js

```diff
@@ -25,6 +25,11 @@ app.get('/swagger.json', (req, res) => {
   res.setHeader('Content-Type', 'application/json');
   res.send(swagger.swaggerSpec);
 });
+
+app.get('/reset-password/:token', (req, res) => {
```

Comment:
line `29` of `index.js` should be in `routes` folder, kindly check the project structure to fix it